draft-ietf-quic-transport.md

```diff
@@ -3720,11 +3720,11 @@ containing that information is acknowledged.
   unless the endpoint has sent a RESET_STREAM for that stream.  Once an endpoint
   sends a RESET_STREAM frame, no further STREAM frames are needed.
 
-* ACK frames carry the most recent set of acknowledgements and the Ack Delay
-  from the largest acknowledged packet, as described in
-  {{sending-acknowledgements}}. Delaying the transmission of packets
-  containing ACK frames or resending old ACK frames can cause the peer to
-  generate an inflated RTT sample or unnecessarily disable ECN.
+* ACK frames carry the most recent set of acknowledgements and the
+  acknowledgement delay from the largest acknowledged packet, as described in
```

Comment:
Ack Delay is the name of the field in the ACK frame, so I thought the old text was correct?

Reply:
It depends on whether the text is supposed to mean that the ACK frame carries the ACK Delay field, or if it carries the acknowledgment delay value. I assumed the latter.

Reply:
yes, in the context of the sentence, `acknowledgement delay` is more appropriate.